pkgs/tools/misc/tere/default.nix

```
@@ -2,20 +2,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "tere";
-  version = "1.2.0";
+  version = "1.3.0";
 
   src = fetchFromGitHub {
     owner = "mgunyho";
     repo = "tere";
     rev = "v${version}";
-    sha256 = "sha256-2DpJYOzswT7dHEIj+qaTx1Jm28NE+61opoxuMChNwbM=";
+    sha256 = "sha256-jousyoro1Mn1+yBzUkGxW7/zbNvF7+Y4/WLRj99Iuy0=";
   };
 
-  cargoSha256 = "sha256-8qSikfsHX8xwBqSSxWwNbiOpXmfzwGTAOALPjxVHadc=";
-
-  # This test confirmed not working.
-  # https://github.com/mgunyho/tere/issues/44
-  cargoPatches = [ ./brokentest.patch ];
+  cargoSha256 = "sha256-hMAxKijmlckkCtQZiC5ubaZQKU2m99gL/MkYoU7zQxU=";
```

this should be able to fix build on aarch64-linux

```suggestion
  cargoSha256 = "sha256-hMAxKijmlckkCtQZiC5ubaZQKU2m99gL/MkYoU7zQxU=";
  
  postPatch = ''
    rm .cargo/config.toml;
  '';
```

---

might be a good idea to have it in your global config